pkg/apis/pubsub/v1alpha1/pullsubscription_types.go

```diff
@@ -56,8 +56,7 @@ var _ = duck.VerifyType(&PullSubscription{}, &duckv1.Conditions{})
 
 // PullSubscriptionSpec defines the desired state of the PullSubscription.
 type PullSubscriptionSpec struct {
-	v1alpha1.PubSubSpec   `json:",inline"`
-	v1alpha1.IdentitySpec `json:",inline"`
```

Comment:
Found I made IdentitySpce redundant here in PR https://github.com/google/knative-gcp/pull/728, PubSubSpec already includes IdentitySpce.